GitUI/Editor/RebaseTodoHighlightingStrategy.cs

```diff
@@ -100,6 +100,18 @@ private static bool TryHighlightInteractiveRebaseCommand(IDocument document, Lin
                             return false;
                         }
 
+                        if (state == State.SpacesAfterCommand)
+                        {
+                            string option = cmd.options.FirstOrDefault(o => index + 1 + o.Length < line.Length && document.GetText(line.Offset + index + 1, o.Length) == o);
+
+                            if (option is not null)
```

Comment:
nit:
```suggestion
                            string? option = cmd.options.FirstOrDefault(o => index + 1 + o.Length < line.Length && document.GetText(line.Offset + index + 1, o.Length) == o);
                            if (option is not null)
```

Reply:
And `string? ...`, please.

Reply:
> I remember reading some blog post which stated that enabling just the annotations is perhaps useless.

Not quite so. We have annotated some projects (where we enforce NRTA) but not the GitUI, where the NRTA aren't enforced:
https://github.com/gitextensions/gitextensions/blob/e8ef6e58b3ecfb1bb6dddcc01dcbb5c7b9af760d/GitUI/GitUI.csproj#L6
You can find more information here: https://learn.microsoft.com/dotnet/csharp/nullable-references#nullable-contexts

Annotating this project requires some effort, and no one so far had free cycles to do that.

Reply:
OK, understood. It's just that I would find it difficult to try annotating the code without any help from the IDE or compiler.